pkg/cluster/spec.go

```diff
@@ -305,7 +255,8 @@ func BuildSpecFromBundles(cluster *eksav1alpha1.Cluster, bundles *v1alpha1.Bundl
 	}
 
 	s.Bundles = bundles
-	s.Cluster = cluster
+	s.Config.Cluster = cluster
+	s.Cluster = s.Config.Cluster
```

Comment:
why do we still need this? is it going to be deprecated in a following PR? If so, can we add //TODO comment saying it's going to be deprecated so folks are aware not using it anymore.

Reply:
We don't need it, I missed this, removing